test/com/xilinx/rapidwright/design/TestSiteRoutingResolver.java

```diff
@@ -0,0 +1,55 @@
+package com.xilinx.rapidwright.design;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.xilinx.rapidwright.device.BELPin;
+import com.xilinx.rapidwright.support.CheckOpenFiles;
+import com.xilinx.rapidwright.support.RapidWrightDCP;
+import com.xilinx.rapidwright.tests.CodePerfTracker;
+import com.xilinx.rapidwright.util.Pair;
+
+public class TestSiteRoutingResolver {
+
+    private Pair<String,String> inputSiteWire1 = new Pair<>("SLICE_X16Y238","A2"); 
+    
+    private Pair<String,String> inputSiteWire2 = new Pair<>("SLICE_X13Y237","F5");
+    
+    @SuppressWarnings("unchecked")
```

Comment:
What warning does this suppress?

Reply:
Creating an array of `Pair<String,String>`.  Not sure why its a problem.

Reply:
Would it be this line `new Pair[] {inputSiteWire1, inputSiteWire2}` perchance? Would `new Pair<String,String>` help? Otherwise I've done `Arrays.asList(...)` in the past too.